plugin/plugins/bilibili_danmaku/danmaku_core.py

```
@@ -648,42 +714,67 @@ async def _connect_once(self):
-            # "no close frame" 是服务器直接断 TCP 的正常情况，降级为 warning
-            if "no close frame" in err_str or "connection closed" in err_str.lower():
-                self._log(f"连接已断开: {err_str}", "warning")
-            else:
-                self._log(f"连接异常: {err_str}", "error")
-                raise
-        finally:
-            if self._heartbeat_task and not self._heartbeat_task.done():
-                self._heartbeat_task.cancel()
-            self._ws = None
-            self._log("弹幕连接已断开")
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)